bindings/python/cconfigspace/__init__.py

```diff
@@ -1,26 +1,68 @@
-import platform
 import ctypes as ct
 import os
+import platform
+import packaging.version
+import warnings
+
+from ctypes.util import find_library
+
+from .__version__ import __version__
+
+
+__ccs_versions___ = [
+  packaging.version.parse("0.0.0"), 
+  packaging.version.parse("0.0.0")
```

Comment:
I've added a VERSION file in the sources that you should be able to get the max value from.